docs/book/src/user/quick-start.md

```diff
@@ -4,735 +4,127 @@ In this tutorial we'll cover the basics of how to use Cluster API to create one
-kind: AWSCluster
-metadata:
-  name: capi-quickstart
-spec:
-  # Change this value to the region you want to deploy the cluster in.
-  region: us-east-1
-  # Change this value to a valid SSH Key Pair present in your AWS Account.
-  sshKeyName: default
-```
-{{#/tab }}
-{{#tab Azure}}
-
-```yaml
-apiVersion: cluster.x-k8s.io/v1alpha3
-kind: Cluster
-metadata:
-  name: capi-quickstart
-spec:
-  clusterNetwork:
-    pods:
-      cidrBlocks:
-      - 192.168.0.0/16
-  infrastructureRef:
-    apiVersion: infrastructure.cluster.x-k8s.io/v1alpha3
-    kind: AzureCluster
-    name: capi-quickstart
----
-apiVersion: infrastructure.cluster.x-k8s.io/v1alpha3
-kind: AzureCluster
-metadata:
-  name: capi-quickstart
-spec:
-  # Change this value to the region you want to deploy the cluster in.
-  location: southcentralus
-  networkSpec:
-    vnet:
-      name: capi-quickstart-vnet
-  # Change this value to the resource group you want to deploy the cluster in.
-  resourceGroup: capi-quickstart
-```
-{{#/tab }}
-{{#tab Docker}}
-
-```yaml
-apiVersion: cluster.x-k8s.io/v1alpha3
-kind: Cluster
-metadata:
-  name: capi-quickstart
-spec:
-  clusterNetwork:
-    pods:
-      cidrBlocks: ["192.168.0.0/16"]
-  infrastructureRef:
-    apiVersion: infrastructure.cluster.x-k8s.io/v1alpha3
-    kind: DockerCluster
-    name: capi-quickstart
----
-apiVersion: infrastructure.cluster.x-k8s.io/v1alpha3
-kind: DockerCluster
-metadata:
-  name: capi-quickstart
-```
-{{#/tab }}
-{{#tab GCP}}
 <aside class="note warning">
 
 <h1>Action Required</h1>
 
-Replace the project below with your unique value and update region if desired.
+If the component specification defined by one of the selected providers expects some environment variables, user
+should ensure those variables are set in advance.
 
 </aside>
 
-```yaml
-apiVersion: cluster.x-k8s.io/v1alpha3
-kind: Cluster
-metadata:
-  name: capi-quickstart
-spec:
-  clusterNetwork:
-    pods:
-      cidrBlocks:
-      - 192.168.0.0/16
-  infrastructureRef:
-    apiVersion: infrastructure.cluster.x-k8s.io/v1alpha3
-    kind: GCPCluster
-    name: capi-quickstart
----
-apiVersion: infrastructure.cluster.x-k8s.io/v1alpha3
-kind: GCPCluster
-metadata:
-  name: capi-quickstart
-spec:
-  project: capi-quickstart
-  region: us-central1
+```shell
+clusterctl init --infrastructure aws
 ```
-{{#/tab }}
-{{#tab vSphere}}
 
-<aside class="note warning">
+The output of `clusterctl init` is similar to this:
 
-<h1>Action Required</h1>
+```shell
+Fetching providers
+Installing cert-manager
+Waiting for cert-manager to be available...
+Installing Provider="cluster-api" Version="v0.3.0" TargetNamespace="capi-system"
+Installing Provider="bootstrap-kubeadm" Version="v0.3.0" TargetNamespace="capi-kubeadm-bootstrap-system"
+Installing Provider="control-plane-kubeadm" Version="v0.3.0" TargetNamespace="capi-kubeadm-control-plane-system"
+Installing Provider="infrastructure-docker" Version="v0.3.0" TargetNamespace="capd-system"
```

Comment:
The command above shows aws, but the output below here shows "infrastructure-docker"